js/app.js

```diff
@@ -87,12 +87,15 @@ function removeExistingImages() {
 
 function renderBusMallImages (item) {
   var busMallImagesParent = document.getElementById('busMallImagesParent');
+  var imgSpan = document.createElement('span');
```

Comment:
I'm just making sure I understand this section correctly, so let me know if/where I'm wrong. You added a <span> element to the busMallImagesParent and added a new class to it called prodImage. Then in CSS, this is the element that will get a transition when hovered over. The span element is created as a container for the image. Is that right?